inst/include/population_dynamics/population/population.hpp

```diff
@@ -211,13 +211,13 @@ struct Population : public FIMSObject<Type> {
         this->mortality_F[i_age_year] +=
             this->fleets[fleet_]->Fmort[year] *
             this->fleets[fleet_]->selectivity->evaluate(ages[age]);
-        FIMS_LOG << " sel age " << ages[age] << "for fleet " << fleet_ << " is "
+        POPULATION_LOG << " sel age " << ages[age] << "for fleet " << fleet_ << " is "
```

Comment:
change " sel age " to " selectivity age " or " selecticity at age "

Reply:
done